apps/admin-gui/src/app/users/pages/user-detail-page/user-dashboard/user-dashboard.component.ts

```diff
@@ -5,10 +5,18 @@ import {
   User,
   UsersManagerService
 } from '@perun-web-apps/perun/openapi';
-import { GuiAuthResolver, NotificatorService, StoreService } from '@perun-web-apps/perun/services';
+import {
+  ApiRequestConfigurationService,
+  GuiAuthResolver,
+  NotificatorService,
+  StoreService
+} from '@perun-web-apps/perun/services';
 import { SideMenuService } from '../../../../core/services/common/side-menu.service';
 import { ActivatedRoute, Router } from '@angular/router';
 import { TranslateService } from '@ngx-translate/core';
+import { getDefaultDialogConfig } from '@perun-web-apps/perun/utils';
+import { MailChangeFailedDialogComponent } from '../../../../../../../../libs/perun/dialogs/src/lib/mail-change-failed-dialog/mail-change-failed-dialog.component';
```

Comment:
I think that all imports from the library should start with `@perun-web-apps/...`